projects/demo/src/modules/components/textfield/examples/5/index.html

```diff
@@ -1,11 +1,23 @@
-<tui-textfield>
-    <strong>π</strong>
-    -value
+<tui-textfield [tuiTextfieldCleaner]="false">
+    <label tuiLabel>Flat angle</label>
 
     <input
+        #input
         placeholder="3,14"
         tuiTextfield
         [maskito]="options"
         [(ngModel)]="value"
     />
+
+    <button
+        *ngIf="value"
+        appearance="icon"
```

Comment:
Note to self: see if textfield should set icon as button default appearance, and xs as size